build.sh

```diff
@@ -852,6 +858,8 @@ case "$COMMAND" in
             cp -R include/Realm include/realm
           fi
           touch include/Realm/RLMPlatform.h
+        else
+          echo "let swiftLanguageVersion = \"$(get_swift_version)\"" > RealmSwift/SwiftVersion.swift
```

Comment:
This definitely needs a comment, but it's because `without-core` is only used when installing Realm Swift.